lib/resources/r4/immunization.yaml

```diff
@@ -65,7 +65,7 @@ fields:
     terminology:
     - display: Immunization Status Reason
       system: http://terminology.hl7.org/CodeSystem/v3-ActReason
```

Comment:
system urls are still- http, should we update them to https. When we navigate to http://terminology.hl7.org/CodeSystem/v3-ActReason ; its redirecting to a new https url, check this.

Also should we carryout this changes for dstu2

Reply:
the "system" value must match the "Official URL" on that page, which has http.